SPECS/abseil-cpp/abseil-cpp.spec

```diff
@@ -11,12 +14,19 @@ Source0:        https://github.com/abseil/abseil-cpp/archive/refs/tags/%{version
 BuildRequires:  cmake >= 3.20.0
 BuildRequires:  gcc
 BuildRequires:  make
-
-%if 0%{?with_check}
-BuildRequires:  gmock >= 1.12.0
-BuildRequires:  gmock-devel >= 1.12.0
-BuildRequires:  gtest >= 1.12.0
-BuildRequires:  gtest-devel >= 1.12.0
+BuildRequires:  ninja-build
+BuildRequires:  gcc-c++
+
+BuildRequires:  gmock
```

Comment:
If these are only needed for ptests, then we probably want to put the "with_check" macro back in place.